### PR TITLE
WIP Added test to cover weblogicLoggingExporterFilters

### PR DESCRIPTION
**Pull request by hzhao-github**
New test passed in http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2335/console

### CODE REVIEW

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java [hunk 1]
```diff
@@ -186,8 +193,8 @@ public void testOperatorLogSearch() throws Exception {
   }
 
   /**
-   * Use Elasticsearch Search APIs to query Weblogic log info. Verify that log hits for Weblogic
-   * servers are not empty
+   * Use Elasticsearch Search APIs to query Weblogic log info. Verify that log hits for 
+   * Weblogic servers are not empty
```
Comment (markxnelson) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: "WebLogic" spelled wrong

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java [hunk 2]
```diff
@@ -251,6 +260,10 @@ public void testWlsLoggingExporter() throws Exception {
     // Verify that hits of _type:doc are not empty
     queryCriteria = "/_search?q=_type:doc";
     verifySearchResults(queryCriteria, regex, wlsIndexKey, false);
+    // Verify that serverName:managed-server1 is filtered out
+    regex = ".*count\":(\\d+),.*failed\":(\\d+)";
```
Comment (markxnelson) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: I would like to see a comment explaining what the regex is meant to do, so it is easier for future developers to understand the intent please.

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: changes made

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: Will add the comments below next time when touching this file again

    // Verify that serverName:managed-server1 is filtered out
    // by checking the count of logs from serverName:managed-server1 is zero and no failures
    // e.g. when running the query:
    // curl -X GET http://elasticsearch.default.svc.cluster.local:9200/wls/_count?q=serverName:managed-server1 
    // Expected return result is:
    // {"count":0,"_shards":{"total":5,"successful":5,"skipped":0,"failed":0}}

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: comments above added in PR: https://github.com/oracle/weblogic-kubernetes-operator/pull/1266

**Pull request merged**